gradle.properties

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# These are global settings for the gradle build process.
+
```

Comment:
Should we enable parallel builds here?

Reply:
Sure, we can enable parallel by default. It's unlikely to have a major impact on the build time, as the most impact is from the singlegoogle-ads subproject and the subprojects are all interdependent. Couldn't hurt though.

See https://docs.gradle.org/current/userguide/performance.html#parallel_execution for some reference.